extension/e2e-tests/addAsset.test.ts

```diff
@@ -81,3 +81,22 @@ test("Adding Soroban verified token", async ({ page, extensionId }) => {
     timeout: 30000,
   });
 });
+test.afterAll(async ({ page, extensionId }) => {
+  if (
+    test.info().status !== test.info().expectedStatus &&
+    test.info().title === "Adding Soroban verified token"
```

Comment:
`Adding Soroban verified token` adds a trustline to the account, so if it fails, we might end up in a state where the account still has the trustline. This `afterAll` removes it